docs/Developers/Git_Basics.rst

```diff
@@ -236,7 +236,8 @@ Before you push the commit, ensure that you are not developing on master again
 by running:
 
 ::
-    git status
+
+    $ git status
```

Comment:
Be consistent, 2 spaces indentation (I've told you that here last time ;) )

Reply:
But the rest of the document (except line 230) uses 4 spaces for indentation

Reply:
Oh indeed, sorry my bad :)